src/hapi/pipelines/utilities/operational_presence.py

```diff
@@ -42,104 +41,103 @@ def __init__(
         self._sector = sector
         self._scraped_data = {}
 
+    # TODO: make this handle all countries once metadata issue is solved
     def run(self):
         reader = self.get_reader()
-        for country in self.datasetinfo:
-            self._scraped_data[country] = []
-            _, iterator = reader.read(self.datasetinfo[country])
-            for row in iterator:
-                newrow = {}
-                for input_header, output_header in zip(
-                    self.datasetinfo[country]["input"],
-                    self.datasetinfo[country]["output"],
-                ):
-                    newrow[output_header] = row[input_header]
-                if newrow in self._scraped_data[country]:
```

Comment:
Will any duplicate rows cause errors later on?

Reply:
oops good catch, I've put it back in!